src/Classes/Particles.ts

```diff
@@ -12,6 +10,10 @@ import {PolygonMaskInlineArrangement} from "../Enums/PolygonMaskInlineArrangemen
  * Particles manager
  */
 export class Particles {
+    public get count() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'count' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)